build.gradle

```diff
@@ -69,13 +69,13 @@ repositories {
 }
 
 dependencies {
-  implementation 'org.sonarsource.scanner.api:sonar-scanner-api:2.16.1.361'
+  implementation 'org.sonarsource.scanner.api:sonar-scanner-api:2.16.2.588'
```

Comment:
this is potentially a functional change, doesn't affect tests only.
Was it necessary for it to run with java 17?
If not, it should be done with a dedicated commit/ticket.

Reply:
yeah indeed, this update might not be neccessary, i will roll it back